diff --git a/node_modules/react-native-tab-view/src/PagerAndroid.js b/node_modules/react-native-tab-view/src/PagerAndroid.js
index 8c1c9e9..81fdeb3 100644
--- a/node_modules/react-native-tab-view/src/PagerAndroid.js
+++ b/node_modules/react-native-tab-view/src/PagerAndroid.js
@@ -1,9 +1,10 @@
 /* @flow */
 
 import * as React from 'react';
-import { View, ViewPagerAndroid, StyleSheet, I18nManager } from 'react-native';
+import { View, StyleSheet, I18nManager } from 'react-native';
 import { PagerRendererPropType } from './PropTypes';
 import type { PagerRendererProps } from './TypeDefinitions';
+import PagerView from '../../react-native-pager-view';
 
 type PageScrollEvent = {
   nativeEvent: {
@@ -45,7 +46,7 @@ export default class PagerAndroid<T: *> extends React.Component<Props<T>> {
   }
 
   _pageChangeCallabck: any;
-  _viewPager: ?ViewPagerAndroid;
+  _viewPager: ?PagerView;
   _isIdle: boolean = true;
   _currentIndex = 0;
 
@@ -151,7 +152,7 @@ export default class PagerAndroid<T: *> extends React.Component<Props<T>> {
     const initialPage = this._getPageIndex(navigationState.index);
 
     return (
-      <ViewPagerAndroid
+      <PagerView
         key={navigationState.routes.length}
         keyboardDismissMode={keyboardDismissMode}
         initialPage={initialPage}
@@ -163,7 +164,7 @@ export default class PagerAndroid<T: *> extends React.Component<Props<T>> {
         ref={el => (this._viewPager = el)}
       >
         {content}
-      </ViewPagerAndroid>
+      </PagerView>
     );
   }
 }
